src/pages/api/homepage.ts

```diff
@@ -27,7 +27,7 @@ const homepageHandler = async (
     });
 
     // 24 hour cache
-    response.setHeader("Cache-Control", "s-maxage=86400");
+    // response.setHeader("Cache-Control", "s-maxage=86400"); //removing bc the featured content is not
```

Comment:
Nitpick: complete the sentence to make it clear why it's removed. Delete instead of comment out as well.